fabbwled-frontend/src/routes/game/Game.tsx

```diff
@@ -1,51 +1,139 @@
 import styles from "./Game.module.scss";
+import { useEffect, useState } from "react";
+import DiceComponent from "../../DiceComponent.tsx";
+import { convertToElement } from "../../SectionRenderer.tsx";
+import { Player, Section } from "../../interfaces/character.ts";
 
 export default function Game() {
+  const [character, setCharacter] = useState<Player>();
+  const [section, setSection] = useState<Section>();
+
+  useEffect(() => {
+    getCharacter().then((player: Player) => {
+      setCharacter(player);
+      getSection(player.currentSection.split("-")[1]).then(setSection);
+    });
+  }, []);
+
+  const getCharacter = async () => {
+    const response = await fetch("http://localhost:8080/api/player");
+    return response.json();
+  };
+
+  console.log(character);
+  function rollDice(diceNumber: number) {
+    console.log(diceNumber);
+  }
+
+  function newGame() {
+    document.cookie = "";
+  }
+
+  function exitGame() {
+    document.cookie = "";
+    window.location.href = "about:blank";
```

Comment:
With window.close() you should be able to close the Tab -> is more sensible imo

```suggestion
   window.close();
```

Reply:
https://stackoverflow.com/questions/19761241/window-close-and-self-close-do-not-close-the-window-in-chrome

Reply:
Aight then its ok